Physics/HelicityFormalism/WignerD.hpp

```diff
@@ -5,10 +5,10 @@
 #ifndef COMPWA_PHYSICS_HELICITY_FORMALISM_WIGNERD_HPP_
 #define COMPWA_PHYSICS_HELICITY_FORMALISM_WIGNERD_HPP_
 
-#include "Core/FunctionTree/FunctionTree.hpp"
 #include "Core/FunctionTree/Functions.hpp"
+#include "Core/FunctionTree/TreeNode.hpp"
 
-#include "qft++/WignerD.h"
+#include "ThirdParty/qft++/include/qft++/WignerD.h"
```

Comment:
Why include like this?

Reply:
We have a slightly customized version of qft++ and its a quite strict part of ComPWA